src/System Application/App/VS Code Integration/src/VsCodeIntegrationImpl.Codeunit.al

```diff
@@ -74,7 +74,7 @@ codeunit 8333 "VS Code Integration Impl."
     var
         DependencyList: TextBuilder;
     begin
-        if NavAppInstalledApp.Find() then
+        if NavAppInstalledApp.FindFirst() then
```

Comment:
```suggestion
        if NavAppInstalledApp.FindSet() then
```

Reply:
FYI, this cop rule should have triggered: 
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/analyzers/codecop-aa0233

I don't know why it did not...